palo_alto_panorama/assets/logs/palo-alto-panorama.yaml

```diff
@@ -0,0 +1,1255 @@
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `risk_of_app` to `palo.alto.panorama.risk_of_app`
+      enabled: true
+      sources:
+        - risk_of_app
+      sourceType: attribute
+      target: palo.alto.panorama.risk_of_app
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `appchar` to `palo.alto.panorama.appchar`
+      enabled: true
+      sources:
+        - appchar
+      sourceType: attribute
+      target: palo.alto.panorama.appchar
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `appcont` to `palo.alto.panorama.appcont`
+      enabled: true
+      sources:
+        - appcont
+      sourceType: attribute
+      target: palo.alto.panorama.appcont
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `appchar` to `palo.alto.panorama.appchar`
+      enabled: true
+      sources:
+        - appchar
+      sourceType: attribute
+      target: palo.alto.panorama.appchar
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `appcont` to `palo.alto.panorama.appcont`
+      enabled: true
+      sources:
+        - appcont
+      sourceType: attribute
+      target: palo.alto.panorama.appcont
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `appsaas` to `palo.alto.panorama.appsaas`
+      enabled: true
+      sources:
+        - appsaas
+      sourceType: attribute
+      target: palo.alto.panorama.appsaas
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `appsubcat` to `palo.alto.panorama.appsubcat`
+      enabled: true
+      sources:
+        - appsubcat
+      sourceType: attribute
+      target: palo.alto.panorama.appsubcat
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `apptech` to `palo.alto.panorama.apptech`
+      enabled: true
+      sources:
+        - apptech
+      sourceType: attribute
+      target: palo.alto.panorama.apptech
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `apprisk` to `palo.alto.panorama.apprisk`
+      enabled: true
+      sources:
+        - apprisk
+      sourceType: attribute
+      target: palo.alto.panorama.apprisk
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
```

Comment:
As said in my facet comment above, if we remove most the facets, we can remove most of these remapper here

Reply:
Yes, with latest message, we should aim to reduce. LEt me know if you need any clarification